src/resources/helpers/utils.js

```diff
@@ -0,0 +1,24 @@
+export const sortResources = (resources) => {
+    return [...resources].sort((a, b) => {
+      if (a.relative_order === null && b.relative_order === null) {
+        return a.resource_name.localeCompare(b.resource_name);
```

Comment:
I had to change `resource_name` to `display_resource_name` to get this to work. Otherwise I get an error `a.resource_name is undefined` when the the `relative_order` values are null.